framework/views/_createTable.php

```diff
@@ -11,9 +11,9 @@
 <?php foreach ($fields as $field):
     if (empty($field['decorators'])): ?>
             '<?= $field['property'] ?>',
-<?php else: ?>
+    <?php else: ?>
```

Comment:
This looks incorrect - it will generate template with incorrect formatting.

Reply:
I think it will work, as the php code is not in the final template, but still, it’s useless change.

Reply:
I'm pretty sure it will add unnecessary tab, which will mess up with indentation of generated migration. In `framework/views/_foreignTables.php` is similar problem, but it affects phpdoc.

Reply:
```
FILE: /Volumes/Code/yii2/framework/views/_createTable.php
-----------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
-----------------------------------------------------------------------------------------------------------------------------
  5 | ERROR | [x] Header blocks must be separated by a single blank line (PSR12.Files.FileHeader.SpacingAfterBlock)
 14 | ERROR | [x] Closing brace indented incorrectly; expected 4 spaces, found 0 (Squiz.WhiteSpace.ScopeClosingBrace.Indent)
-----------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-----------------------------------------------------------------------------------------------------------------------------
```

Reply:
I think that generating correct output is much more important than following some CS rules, so I suggest to disable views from CS validation/fixing.

Reply:
fixed in [5b9eb7d](https://github.com/yiisoft/yii2/pull/20121/commits/5b9eb7df339296163905bd0c3bf083f9c08ea752)